File: bindings/go/binding_test.go
package tree_sitter_lilypond_test

import (
	"testing"

	tree_sitter "github.com/tree-sitter/go-tree-sitter"
	tree_sitter_lilypond "github.com/tree-sitter/tree-sitter-lilypond/bindings/go"
)

func TestCanLoadLilyPondGrammar(t *testing.T) {
	language := tree_sitter.NewLanguage(tree_sitter_lilypond.LanguageLilyPond())
	if language == nil {
		t.Errorf("Error loading LilyPond grammar")
	}
}

func TestCanLoadLilyPondSchemeGrammar(t *testing.T) {
	language := tree_sitter.NewLanguage(tree_sitter_lilypond.LanguageLilyPondScheme())
	if language == nil {
		t.Errorf("Error loading Lilypond Scheme grammar")
	}
}

File: bindings/node/binding_test.js
/// <reference types="node" />

const assert = require("node:assert");
const { test } = require("node:test");

const Parser = require('tree-sitter');

test("can load LilyPond grammar", () => {
  const parser = new Parser();
  assert.doesNotThrow(() => parser.setLanguage(require("./lilypond")));
});

test("can load LilyPond Scheme grammar", () => {
  const parser = new Parser();
  assert.doesNotThrow(() => parser.setLanguage(require("./lilypond-scheme")));
});
